<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AServer</class>
 <widget class="QMainWindow" name="AServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>接收端</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/transfer/res/transfer/fileTrans.png</normaloff>:/transfer/res/transfer/fileTrans.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="fileListContainer">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="fileListGroup">
          <property name="title">
           <string>文件列表</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QListWidget" name="fileListWidget">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="transferDetailsContainer">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="transferDetailsGroup">
          <property name="title">
           <string>传输详情</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWidget" name="detailsWidget" native="true">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="statusLabel">
                <property name="text">
                 <string>状态：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="statusValue">
                <property name="text">
                 <string>等待连接...</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="currentFileLabel">
                <property name="text">
                 <string>当前文件：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="currentFileValue">
                <property name="text">
                 <string>无</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="sizeLabel">
                <property name="text">
                 <string>文件大小：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="sizeValue">
                <property name="text">
                 <string>0 B</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="speedLabel">
                <property name="text">
                 <string>传输速度：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="speedValue">
                <property name="text">
                 <string>0 B/s</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="remainingTimeLabel">
                <property name="text">
                 <string>剩余时间：</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="remainingTimeValue">
                <property name="text">
                 <string>--:--:--</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QProgressBar" name="progressBar">
                <property name="value">
                 <number>0</number>
                </property>
                <property name="format">
                 <string>%p%</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionListen"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionSetSavePath"/>
   <addaction name="actionOpenSavePath"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionListen">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/transfer/res/transfer/tranStart.png</normaloff>:/transfer/res/transfer/tranStart.png</iconset>
   </property>
   <property name="text">
    <string>开始监听</string>
   </property>
   <property name="toolTip">
    <string>等待发送方连接</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/transfer/res/transfer/disConnect.png</normaloff>:/transfer/res/transfer/disConnect.png</iconset>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
   <property name="toolTip">
    <string>断开与发送方的连接</string>
   </property>
  </action>
  <action name="actionSetSavePath">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/transfer/res/transfer/save.png</normaloff>:/transfer/res/transfer/save.png</iconset>
   </property>
   <property name="text">
    <string>设置保存路径</string>
   </property>
   <property name="toolTip">
    <string>设置接收文件的保存位置</string>
   </property>
  </action>
  <action name="actionOpenSavePath">
   <property name="icon">
    <iconset>
     <normaloff>:/transfer/res/transfer/folder.png</normaloff>:/transfer/res/transfer/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开保存位置</string>
   </property>
   <property name="toolTip">
    <string>在文件管理器中打开保存位置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/transfer/res/transfer/help.png</normaloff>:/transfer/res/transfer/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>查看使用说明</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
